### PR TITLE
Update SMS Rates

### PR DESCRIPTION
**Pull request by gcapalbo**
Updates SMS Rates for two new backends

@nickpell 
@sravfeyn code buddy
@emord fyi on icds backend rates

### CODE REVIEW

File: corehq/apps/smsbillables/management/commands/bootstrap_apposit_gateway.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+from django.core.management.base import BaseCommand
+from corehq.apps.accounting.models import Currency
+from corehq.apps.sms.models import INCOMING, OUTGOING
+from corehq.apps.smsbillables.models import SmsGatewayFee, SmsGatewayFeeCriteria
+from corehq.apps.smsbillables.utils import log_smsbillables_info
+from corehq.messaging.smsbackends.apposit.models import SQLAppositBackend
+
+
+def bootstrap_apposit_gateway(apps=None):
+    usd_currency, _ = (apps.get_model('accounting', 'Currency') if apps else Currency).objects.get_or_create(code="USD")
```
Comment (dimagimon) [inline corehq/apps/smsbillables/management/commands/bootstrap_apposit_gateway.py]: E501 line too long (120 > 115 characters)

File: corehq/apps/smsbillables/management/commands/bootstrap_apposit_gateway.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    sms_gateway_fee_class = apps.get_model('smsbillables', 'SmsGatewayFee') if apps else SmsGatewayFee
+    sms_gateway_fee_criteria_class = apps.get_model('smsbillables', 'SmsGatewayFeeCriteria') if apps else SmsGatewayFeeCriteria
```
Comment (dimagimon) [inline corehq/apps/smsbillables/management/commands/bootstrap_apposit_gateway.py]: E501 line too long (127 > 115 characters)

File: corehq/apps/smsbillables/management/commands/bootstrap_icds_gateway.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+from django.core.management.base import BaseCommand
+from corehq.apps.accounting.models import Currency
+from corehq.apps.sms.models import INCOMING, OUTGOING
+from corehq.apps.smsbillables.models import SmsGatewayFee, SmsGatewayFeeCriteria
+from corehq.apps.smsbillables.utils import log_smsbillables_info
+from corehq.messaging.smsbackends.icds.models import SQLICDSBackend
+
+
+def bootstrap_icds_gateway(apps=None):
+    usd_currency, _ = (apps.get_model('accounting', 'Currency') if apps else Currency).objects.get_or_create(code="USD")
```
Comment (dimagimon) [inline corehq/apps/smsbillables/management/commands/bootstrap_icds_gateway.py]: E501 line too long (120 > 115 characters)

File: corehq/apps/smsbillables/management/commands/bootstrap_icds_gateway.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    sms_gateway_fee_class = apps.get_model('smsbillables', 'SmsGatewayFee') if apps else SmsGatewayFee
+    sms_gateway_fee_criteria_class = apps.get_model('smsbillables', 'SmsGatewayFeeCriteria') if apps else SmsGatewayFeeCriteria
```
Comment (dimagimon) [inline corehq/apps/smsbillables/management/commands/bootstrap_icds_gateway.py]: E501 line too long (127 > 115 characters)

File: corehq/apps/smsbillables/management/commands/bootstrap_apposit_gateway.py [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+    SmsGatewayFee.create_new(
+        SQLAppositBackend.get_api_id(),
+        INCOMING,
+        Decimal('0.02'),
+        currency=usd_currency,
```
Comment (gcapalbo) [inline corehq/apps/smsbillables/management/commands/bootstrap_apposit_gateway.py]: @nickpell this is the currency that this rate is quoted in, right? a lot of the other bootstrap gateway commands were using settings.DEFAULT_CURRENCY for this, but it seems like we should pin it to USD if that's the case

**Review by emord — APPROVED** [review]

File: corehq/apps/smsbillables/management/commands/bootstrap_apposit_gateway.py (hunk 5 above)
Comment (nickpell) [inline corehq/apps/smsbillables/management/commands/bootstrap_apposit_gateway.py]: yeah that does sound right.

File: corehq/apps/smsbillables/management/commands/bootstrap_icds_gateway.py [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+    SmsGatewayFee.create_new(
+        SQLICDSBackend.get_api_id(),
+        INCOMING,
+        Decimal('0.0'),
```
Comment (nickpell) [inline corehq/apps/smsbillables/management/commands/bootstrap_icds_gateway.py]: 👍

**Review by nickpell — APPROVED** [review]

**Comment by gcapalbo** [comment]
(approved and tests passed, merging)

**Pull request merged**